gematria/datasets/block_wrapper.h

```diff
@@ -17,33 +17,42 @@
 
 #include <stdint.h>
 
-#include "absl/types/span.h"
+#include "third_party/absl/types/span.h"
```

Comment:
This change should probably be reveerted?

Reply:
Ah whoops, missed that one.